docs/docs/running-on-minikube-without-building.md

```diff
@@ -0,0 +1,294 @@
+
+-   k9s
+
+## Preparation
+
+### Starting Kubernetes
+
+```bash
+minikube config set driver hyperkit
+
+# Currently supported by Magda
+minikube config set kubernetes-version v1.16.5
+minikube config set memory 16384
+minikube config set cpus 4
+minikube config set dashboard true
+minikube config set disk-size 60G
+minikube start
+```
+
+### Starting Magda with the [Building-and-Running](https://magda.io/docs/building-and-running) instructions \*\*
+
+\*\* Minus the building part.
+
+Install the prerequisites on the Building-and-Running page.
+Then:
+
+```bash
+helm repo add magda-io https://charts.magda.io
+helm repo add twuni https://helm.twun.io
+# Get update from repos
+helm repo update
+```
+
+Those instructions are missing these steps (but I guess they're
+implied):
+
+```bash
+git clone https://github.com/magda-io/magda.git
+cd magda
+```
+
+Then, following the instructions:
+
+```bash
+helm install docker-registry -f deploy/helm/docker-registry.yml twuni/docker-registry
+helm repo add mittwald https://helm.mittwald.de
+helm repo update
+kubectl create namespace kubernetes-replicator
+helm upgrade --namespace kubernetes-replicator --install kubernetes-replicator mittwald/kubernetes-replicator
+```
+
+Edit the file at `deploy/helm/minikube-dev.yaml` to comment out these lines:
+
+```yaml
+# image:
+#   repository: "localhost:5000/data61"
+#   tag: "latest"
+#   pullPolicy: Always
+```
+
+Then create secrets (and, for output see the bottom of this file) :
+
+```bash
+nvm use lts/erbium
+npm install --global @magda/create-secrets
+export CLUSTER_NAMESPACE=magda
+create-secrets
+```
+
+Then:
+
+```bash
+# update magda helm repo
+helm repo update
+
+# update magda chart dependencies
+yarn update-all-charts
+
+# deploy the magda chart from magda helm repo
+kubectl create namespace magda
+helm upgrade --install --timeout 9999s -n magda -f deploy/helm/minikube-dev.yml magda deploy/helm/local-deployment
+```
+
+I suggest using `k9s` to monitor the start-up of Magda. Due to my configuration,
+the `magda-auth` pods and the `correspondence-api` pod don't start properly, but that
+is to be expected at this stage.
+
+### SSL/HTTPS access to Magda
+
+I would like to have https access, so following the link from the Building-and-Running page to
+[here](https://magda.io/docs/how-to-setup-https-to-local-cluster.html):
+
+```bash
+kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.crds.yaml
+
+# If you haven't install jetstack helm chart repo yet
+helm repo add jetstack https://charts.jetstack.io
+
+# Create namespace for cert-manager
+kubectl create namespace cert-manager
+
+# If you use Helm v3+
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v1.0.4
+
+kubectl apply -f https://gist.githubusercontent.com/t83714/51440e2ed212991655959f45d8d037cc/raw/7b16949f95e2dd61e522e247749d77bc697fd63c/selfsigned-issuer.yaml
+
+minikube addons enable ingress
```

Comment:
Probably move `minikube addons enable ingress` line up (above first --- line 112).
This command will actually install the ngix ingress controller to `kubectl-system` namespace --- may worth run it earlier